pandas/tests/indexes/test_multi.py

```diff
@@ -3066,8 +3066,7 @@ def test_nan_stays_float(self):
         assert pd.isna(idx0.get_level_values(1)).all()
         # the following failed in 0.14.1
         assert pd.isna(idxm.get_level_values(1)[:-1]).all()
-
-        df0 = pd.DataFrame([[1, 2]], index=idx0)
```

Comment:
what is the issue here?

Reply:
the (outer) list has a different length than the index